clifford/test/test_clifford.py

```diff
@@ -29,6 +29,16 @@ def test_inverse(self, algebra):
                 assert abs((a_inv * a)-1.) < 1.e-11
                 assert abs(a_inv - 1./a) < 1.e-11
 
+    def test_pseudoscalar(self, algebra):
+        """
+        Check if the outer product of the basis elements
+        is a linear multiple of the pseudoscalar
+        """
+        layout, blades = algebra
+        ps = MVArray([kv[1] for kv, i in zip(blades.items(), range(layout.dims))]).op()
```

Comment:
```suggestion
        ps = MVArray(layout.blades_of_grade(1)).op()
```
or
```suggestion
        ps = functools.reduce(clifford.op, layout.blades_of_grade(1))
```
Although you'll need #155 merged for either to work

Reply:
(which it now is)